org.opentosca.yamlconverter.main/src/main/java/org/opentosca/yamlconverter/main/utils/FileUtil.java

```diff
@@ -17,7 +17,7 @@
 	 * @throws IOException
 	 */
 	public String readYamlResource(String path) throws URISyntaxException, IOException {
-		final File simpleYaml = new File(getClass().getResource(path).toURI());
+		final File simpleYaml = new File(path);
```

Comment:
Then you can also make this static, so it is not neccessary to instantiate FileUtil.